xcode.md

```diff
@@ -299,6 +300,7 @@ Got access to old versions of Apple installers you know are legit? Submit some h
  [13.0 beta 2]: https://download.developer.apple.com/Developer_Tools/Xcode_13_beta_2/Xcode_13_beta_2.xip
  [13.0 beta 3]: https://download.developer.apple.com/Developer_Tools/Xcode_13_beta_3/Xcode_13_beta_3.xip
  [13.0 beta 5]: https://download.developer.apple.com/Developer_Tools/Xcode_13_beta_5/Xcode_13_beta_5.xip
+ [13.0 RC]: https://download.developer.apple.com/Developer_Tools/Xcode_13_Release_Candidate/Xcode_13_Release_Candidate.xip
```

Comment:
You’ve borrowed this reference and URL for 13.0-final-not-RC, but it’s not correct for that purpose.